examples/dem/3d-silo/silo-Golshan.prm

```diff
@@ -17,19 +18,26 @@ end
 
 subsection restart
   set checkpoint = true
-  set filename   = sliding_restart
```

Comment:
Why did you remove it? Why not just change the name?

Reply:
I think it's better to leave the default restart file name...